docs/snapshotting/versioning.md

```diff
@@ -88,16 +118,18 @@ new Firecracker process.
 
 ### CPU model
 
-Firecracker microVMs can run on any Intel x86_64 CPU models that support the hardware virtualizations extensions. Snapshots, however, are not compatible across all Intel x86_64 CPU models. They are only compatible if the CPU features exposed to the guest are an invariant when saving and restoring the snapshot.
-The trivial scenario is creating and restoring snapshots on hosts that have the same CPU model. 
-To relax this limitation, Firecracker provides an API to select a CPU template. Firecracker CPU templates work by using CPUID masking that restricts the exposed features to a common denominator of multiple CPU models.
+Firecracker micromVMs can run on Intel/AMD cpu models that support the hardware virtualizations extensions. Snapshot are not compatible across CPU architectures and even across CPU models of the same architecture. They are only compatible if the CPU features exposed to the guest are an invariant when saving and restoring the snapshot. The trivial scenario is creating and restoring snapshots on hosts that have the same CPU model. 
+
+To make snapshots more portable across Intel CPUs Firecracker provides an API to select a CPU template which is only available for Intel - T2 and C3. There are no templates available for AMD. Firecracker CPU templates mask CPUID to restrict the exposed features to a common denominator of multiple CPU models. 
```

Comment:
are these templates related in any way to AWS's instance types?

Reply:
Yes, I will clarify.